main.py

```diff
@@ -2,6 +2,286 @@
 # from visma.gui.cli import commandExec
 
 
+from PyQt5.QtGui import QPainter
```

Comment:
The lines 5-284 are redundant as these are already present in the visma/gui/window.py file. Hence the cause for build fail.

Reply:
Oh. That was a mistake.